packages/markdown/src/lib/remark-slate/remarkPlugin.ts

```diff
@@ -12,5 +14,7 @@ export function remarkPlugin(options: RemarkPluginOptions) {
   };
 
   // @ts-ignore
-  this.Compiler = compiler;
+  this.Compiler = options.editor.getOptions(MarkdownPlugin).keepLineBreak
+    ? (node: MdastNode) => remarkLineBreakCompiler(node, options)
+    : compiler;
```

Comment:
The `@ts-ignore` was fine before when the line was a simple assignment, but it's a bit more dangerous now that the code is more complex. Can you refactor the logic that chooses the compiler to be type safe, such as by storing it in a variable before the `@ts-ignore` line?

Also, it looks like `compiler` is completely unused if the option is true. Is that intended?

Reply:
Sorry for the late reply, I'll give it a try

Reply:
This compiler is the default one before, and it exists in this function. It is true that this function will be repeatedly generated every time it is executed, so do I need to extract it separately as a single file here?

Reply:
I don't think it matters how often this particular function is created, so long as the code for the function is type safe (so that we can automatically detect a type error if the signature of `getOptions(MarkdownPlugin)` or `remarkLineBreakCompiler` changes).

Why is the previous `compiler` variable unused if `keepLineBreak` is true?

Reply:
Sorry for the late solution. I have submitted a new commit now. Could you please check if it is as expected?